### ISSUE TITLE
Register Page Not Completed on Mern Ecommerce App

### ISSUE THREAD

**Issue by Saswat689** [safak/youtube#50]
I was following your series of mern ecommerce app and saw that you missed out the Register Page.

My questions are:

- Is there any valid reason for doing it.
- If yes then please inform us.

It will be easy for us to create the Register page cuz its similar to login. But do we need to ?

**Comment by freddyfernandes** [comment]
yeah, the register page is just a blank form with no functionality. Have you implemented the backend connectivity for it?

**Comment by freddyfernandes** [comment]
> I was following your series of mern ecommerce app and saw that you missed out the Register Page.
> 
> My questions are:
> 
> * Is there any valid reason for doing it.
> * If yes then please inform us.
> 
> It will be easy for us to create the Register page cuz its similar to login. But do we need to ?

yeah, the register page is just a blank form with no functionality. Have you implemented the backend connectivity for it?

**Comment by Saswat689** [comment]
@freddyfernandes I think this is what you are looking for [Auth Route](https://github.com/Saswat689/ecom_backend/blob/master/routes/auth.js)

If you need help with any client files reach out to my other repositories

Cheers,
Saswat
